arch/arm/src/stm32h7/stm32_bbsram.c

```diff
@@ -470,7 +470,7 @@ static off_t stm32_bbsram_seek(struct file *filep, off_t offset,
    *   point, subsequent reads of data in the gap shall return bytes with the
    *   value 0 until data is actually written into the gap."
    *
-   * We can conform to the first part, but not the second.  But return EINVAL
+   * We can conform to the first part, but not the second. But return -EINVAL
```

Comment:
```suggestion
   * We can conform to the first part, but not the second.  But return -EINVAL
```

Reply:
The length will exceed the limitation of 79/line.